src/gpo/assets/PowerToys.admx

```diff
@@ -495,6 +495,16 @@
         <decimal value="0" />
       </disabledValue>
     </policy>
+    <policy name="AllowDiagnosticData" class="Both" displayName="$(string.AllowDiagnosticData)" explainText="$(string.AllowDiagnosticDataDescription)" key="Software\Policies\PowerToys" valueName="AllowDataDiagnostics">
+      <parentCategory ref="GeneralSettings" />
+      <supportedOn ref="SUPPORTED_POWERTOYS_0_84_0" />
```

Comment:
If the PR gets not in for the 0.84.0 release please update the `supportedOn` value for the next release and do not forget to update the admx version/revision too.